http-protocol-binding.md

```diff
@@ -439,6 +445,119 @@ Content-Length: nnnn
 
 ```
 
+### 3.4. Multipart Content Mode
+
+In the _multipart_ content mode several events are sent into a single HTTP multipart
+request or response as described in [RFC2046, section 5.1][rfc2046-section-5.1]. 
+The _multipart_  content mode maps one event per part. Because each part consists in an 
+group of headers and a body, an event could be encoded following the same rules that applies
+to [Structured Content Mode](#32-structured-content-mode) and [Binary Content Mode](#31-binary-content-mode)
+within the part, respectively named _structured part_ and _binary part_.
+
+Every part of the HTTP envelope MUST be or a _structured part_ or a _binary part_.
+
+#### 3.4.1. HTTP Content-Type
+
+The [HTTP `Content-Type`][content-type] header MUST be set to the media type of `multipart/cloudevents` 
+and must include the [boundary parameter](https://tools.ietf.org/html/rfc2046#section-5.1.1)
+
+```text
+Content-Type: multipart/cloudevents; boundary=12345
+```
+
+#### 3.4.2. Structured Part
+
+All rules defined in [Structured Content Mode](#32-structured-content-mode) applies from the beginning of the 
```

Comment:
I think there needs to be a MUST in here someplace.

Reply:
reworked the sentence, check it now